client/landing/jetpack-cloud/sections/backups/rewind-flow/download.tsx

```diff
@@ -0,0 +1,190 @@
+import getBackupDownloadId from 'state/selectors/get-backup-download-id';
+import getBackupDownloadUrl from 'state/selectors/get-backup-download-url';
+import getBackupDownloadProgress from 'state/selectors/get-backup-download-progress';
+import ProgressBar from './progress-bar';
+import QueryRewindBackupStatus from 'components/data/query-rewind-backup-status';
+import RewindConfigEditor from './rewind-config-editor';
+
+interface Props {
+	rewindId: string;
+	siteId: number;
+}
+
+const BackupDownloadFlow: FunctionComponent< Props > = ( { rewindId, siteId } ) => {
+	const dispatch = useDispatch();
+	const moment = useLocalizedMoment();
+	const translate = useTranslate();
+
+	const [ rewindConfig, setRewindConfig ] = useState< RewindConfig >( defaultRewindConfig );
+
+	const downloadId = useSelector( state => getBackupDownloadId( state, siteId, rewindId ) );
+	const downloadUrl = useSelector( state => getBackupDownloadUrl( state, siteId, rewindId ) );
+	const downloadProgress = useSelector( state =>
+		getBackupDownloadProgress( state, siteId, rewindId )
+	);
+
+	const downloadTimestamp = moment.unix( rewindId ).format( 'LLL' );
+
+	const requestDownload = useCallback(
+		() => dispatch( rewindBackup( siteId, rewindId, rewindConfig ) ),
+		[ dispatch, rewindConfig, rewindId, siteId ]
+	);
+
+	const renderConfirm = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-ready.svg"
+					alt="jetpack cloud download ready"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">{ translate( 'Create downloadable backup' ) }</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'{{strong}}%(downloadTimestamp)s{{/strong}} is the selected point to create a download backup of. ',
+					{
+						args: {
+							downloadTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<h4 className="rewind-flow__cta">{ translate( 'Choose the items you wish to restore:' ) }</h4>
+			<RewindConfigEditor currentConfig={ rewindConfig } onConfigChange={ setRewindConfig } />
+			<RewindFlowNotice
+				gridicon="notice-outline"
+				title={ translate( 'More info' ) }
+				type={ RewindFlowNoticeLevel.NOTICE }
+				link="https://jetpack.com/support/backup"
+			/>
+			<Button
+				className="rewind-flow__primary-button"
+				primary
+				onClick={ requestDownload }
+				disabled={ Object.values( rewindConfig ).every( setting => ! setting ) }
+			>
+				{ translate( 'Create downloadable file' ) }
+			</Button>
+		</>
+	);
+
+	const renderInProgress = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-ready.svg"
+					alt="jetpack cloud download ready"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">
+				{ translate( 'Currently creating a downloadable backup of your site' ) }
+			</h3>
+			<ProgressBar percent={ downloadProgress } />
```

Comment:
Since this ProgressBar is only used once do we need to make it a component?

Reply:
> Add new Jetpack Cloud RewindFlow component to be used for both restore and download flow
I will circle back around and reuse it for the nearly-identical restore flow